words_dir/tweet_nouns.py

```diff
@@ -29,6 +29,7 @@
     'capybara',
     'cat 🐈',
     'cat',
+    'catfish ,
```

Comment:
Please add a closing quote. If you don't, this line and the next will get concatenated or smashed together to be one word